state-chain/custom-rpc/src/lib.rs

```diff
@@ -507,8 +507,8 @@ pub trait CustomApi {
 		at: Option<state_chain_runtime::Hash>,
 	) -> RpcResult<Option<FailingWitnessValidators>>;
 
-	#[method(name = "decode_runtime_event")]
-	fn cf_decode_runtime_events(&self, events: Vec<String>) -> RpcResult<Vec<String>>;
+	#[method(name = "block_event")]
+	fn cf_get_events(&self, at: Option<state_chain_runtime::Hash>) -> RpcResult<Vec<String>>;
```

Comment:
I think the result should be `Vec<serde_json::Value>`. The rpc framework will convert it to a json string. 

This is why all the outputs are escaped: 

```json
{
   "jsonrpc":"2.0",
   "result":[
      "{\"name\":\"EthereumThresholdSigner\",\"values\":[{\"name\":\"KeygenSuccess\",\"values\":[1]}]}",
	]
}
```

instead of:

```json
{
   "jsonrpc":"2.0",
   "result":[
      {"name":"EthereumThresholdSigner","values":[{"name":"KeygenSuccess","values":[1]}]},
	]
}
```